concepts/0020-message-types/README.md

```diff
@@ -29,124 +29,13 @@ Types are specified within a message using the `@type` attribute:
-  did-service        = 1*servchar *( ";" 1*servchar )
-  servchar           = idchar / "=" / "&"
-```
-
-The purpose of the `did-service` component that may optionally follow a DID is to enable construction of a DID reference that may be algorithmically transformed into a concrete URL to access the target resource. There are two algorithms for this transformation. Both begin with the same first step:
-
-1. Extract the DID plus the `did-service` component from the DID reference. Call this the *DID service locator*. Call the rest of the DID reference the `service-specific` string.
-
-#### Service Selection by ID
-
-This algorithm MUST be used if the `did-service` component does NOT begin with the string `type=`.
-
-2. Select the first `service` object whose id property contains an exact match to the DID service locator.
-3. Select the `serviceEndpoint` property of the selected `service` object.
-4. Extract the value of the `serviceEndpoint` property. Call this the *endpoint URL*.
-5. Append the service-specific string to the endpoint URL.
-6. The final result is the concrete URL.
-
-##### Example
-
-The following agent message is received with a type not known to the developer:
-
-```json
-{
-    "@type": "did:sov:123456789abcdefghi1234;spec/exampleprotocol/1.0/exampletype",
-    "attr_a": 5,
-    "attr_b": "Gouda"
-}
-```
-
-The `@type` is extracted, with a DID reference that resolves to the example `service` block above:
-
-``` json
-did:sov:123456789abcdefghi1234;spec/exampleprotocol/1.0/exampletype
-```
-
-A DID resolver would algorithmically transform that DID reference to the following concrete URL:
-
-```
-https://sovrin.org/specs/exampleprotocol/1.0/exampletype
-```
-
-The developer would then be able to load this URL in a browser to discover the meaning of `attr_a` and `attr_b`.
+Message types are URIs that may resolve to developer documentation for the message type, as described in [Protocol URIs](../0003-protocols/README.md#message-type-and-protocol-identifier-uris). We recommend that message type URIs be HTTP URLs.
 
 ### Aries Core Message Namespace
 
-> The Aries community is currently in the process of changing the prefix for protocol message types that currently use the did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/ to use https://didcomm.org/. This proces is described in RFC 0348 [Transition Message Type to HTTPs](../../features/0348-transition-msg-type-to-https/README.md).
-
-`did:sov:BzCbsNYhMrjHiqZDTUASHg` will be used to namespace protocols defined by the community as "core protocols" or protocols that agents should minimally support.
+`https://didcomm.org/` is used to namespace protocols defined by the community as "core protocols" or protocols that agents should minimally support.
 
-This DID is currently held by Daniel Hardman. Ownership will be transferred to the correct entity as soon as possible.
+The `didcomm.org` [DNS entry](https://whois.whoisxmlapi.com/lookup-report/zjRXrYwV5r) is currently controlled by Daniel Hardman.
```

Comment:
Control has now been transferred to DIF (about 2 weeks ago).

Reply:
Thanks - I'll update.

Reply:
Corrected with links to DIF and the DIDComm Messaging spec.

Reply:
Thanks for maintaining control of that and transferring to DIF as well @dhh1128 up until this point.